### DISCUSSION TITLE
How to find config.js?

### DISCUSSION THREAD

**Discussion by viviantic** [l3atbc-datadog/pushkin#108] [General] [answered]
Hello Pushkin team! 

I am very new to coding and I have a baby-level question. I was able to follow the instructions on the Quickstart page until the section about "Setting Up Logins." Could you elaborate a little bit on the instruction below, how to find "config.js," and how to set the useAuth value? 

"In config.js, located at ./pushkin/front-end/src, set useAuth to true or false depending on whether you want to have a login system or not. Note that you cannot use a forum without a login system:
useForum: false,
useAuth: false," 
(https://languagelearninglab.gitbook.io/pushkin/getting-started/quickstart#setting-up-logins)

Thank you very much.

**Comment by jkhartshorne** [comment, marked as answer]
I think you are confused as to how to read `./pushkin/front-end/src`.

This means that you should look at the folder you have your experiment installed in. There should be a folder in it called `pushkin`. That should have a folder called `front-end`. That should have a folder called `src`. And that should have a file called `config.js`. Open and edit that file.

Please let me know if you are still confused.

**Reply by jkhartshorne** [reply]
PS You may find it useful to get a little introduction to the command line. Here's an [introductory tutorial](https://pythonise.com/series/learning-linux/a-gentle-introduction-to-bash).

**Reply by viviantic** [reply]
Awesome! Thank you very much!
